tests/test_functons.py

```diff
@@ -0,0 +1,47 @@
+from codecademy.functions import one_good_turn
+from codecademy.functions import square
+from codecademy.functions import deserves_another
+from codecademy.functions import biggest_number
+
+import pytest, sys
```

Comment:
Once @09041985 and @nicolas13sochi get a grip on general programming, i will intoduce a PEP8 check as part of the Merge Request automatic checks